Tidy up and readme

trombik opened the pull request:
no functionality change

Commit:
initial support of Ubuntu

Commit:
initial support of OpenBSD

Commit:
update meta

Commit:
s/nfsd_exports_file/__nfsd_exports_file/

Commit:
add a handler to debug build in Jenkins

Commit:
add a comment why `rc_stop` is `NO`

it is deliverately disabled

Commit:
try `retries`

Commit:
fix `until`

Commit:
document `retries`

in Jenkins job, the handler fails a few times, succeeds after second or
third try.

Commit:
remove unused variables

and rename some variables in non-standard notation

Commit:
use nfs_server_*_flags in OpenBSD

Commit:
s/nfsd_exports/nfs_server_exports/g

Commit:
rename non-standard-convention variables

Commit:
move common variables on top

Commit:
bugfix: trivial space breaks idempotence test

Commit:
define platform's default in var/*.yml

`-tun 4` is not the default for `nfsd(8)` but is what most users expect
as default

Commit:
QA

Commit:
update README

Commit:
use `Restart mountd OpenBSD`

and put the comment in the right place.
https://github.com/reallyenglish/ansible-role-nfs-server/pull/4#discussion_r121040226

Commit:
Merge branch 'support_other_platforms' into tidy_up_and_readme

Commit:
Merge branch 'master' into tidy_up_and_readme

mitsururike commented on README.md:
```diff
@@ -1,16 +1,80 @@
+| Variable | Description | Default |
+|----------|-------------|---------|
+| `nfs_server_exports_file` | path to `exports(5)`                          | `{{ __nfs_server_exports_file }}` |
+| `nfs_server_exports`      | list of settings of exported path (see below) | `[]` |
```
(see below) but no description.

Commit:
add `nfs_server_exports`

https://github.com/reallyenglish/ansible-role-nfs-server/pull/6#discussion_r121053370
